### PR TITLE
Add Netgate 8200-max devicetype

### PR DESCRIPTION
**Pull request by jnicholsDLR**

### CODE REVIEW

File: device-types/Netgate/8200-max.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      or XON/OFF
+  - name: console-rj45
+    type: rj-45
+    description: |
+      Auto-detect console port: use RJ-45 -OR- micro-USB.
+      115200 baud rate, 8 data bits, no parity, 1 stop bit, no flow control
+      or XON/OFF
+power-ports:
+  - name: psu1
+    type: iec-60320-c6
```
Comment (danner26) [inline device-types/Netgate/8200-max.yaml:27]: The power type of the device itself is `dc-terminal`. We are not modelling the power brick here

Please adjust accordingly

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Netgate/8200-max.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: Netgate
+model: 8200-max
```
Comment (danner26) [inline device-types/Netgate/8200-max.yaml:3]: It looks like the proper name of this is `8200 Max PFSense+ Security Gateway`

File: device-types/Netgate/8200-max.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+slug: netgate-8200-max
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Netgate/8200-max.yaml:6]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Netgate/8200-max.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+console-server-ports:
+  - name: usb1
+    type: usb-a
+  - name: usb2
+    type: usb-a
```
[the remark this answers is not in the document]
Comment (danner26) [reply device-types/Netgate/8200-max.yaml:11]: Okay then this is fine

File: device-types/Netgate/8200-max.yaml (hunk 2 above)
Comment (jnicholsDLR) [reply device-types/Netgate/8200-max.yaml]: addressed in [58623ef](https://github.com/netbox-community/devicetype-library/pull/1709/commits/58623ef29ea86f1fc374c6f6ace75ceacd013f7a)

File: device-types/Netgate/8200-max.yaml (hunk 3 above)
Comment (jnicholsDLR) [reply device-types/Netgate/8200-max.yaml]: `airflow` has been added in [58623ef](https://github.com/netbox-community/devicetype-library/pull/1709/commits/58623ef29ea86f1fc374c6f6ace75ceacd013f7a)

I do not have enough information to supply the `weight` and `weight_unit` fields.

**Pull request closed without merging**